Remotion/Validation/Core/Validators/LengthValidator.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace Remotion.Validation.Validators
 {
-  public class LengthValidator : ILengthValidator
+  public class LengthValidator : IMinimumLengthValidator, IMaximumLengthValidator
```

Comment:
please unify the protected ctor into the public ctor. Max is no longer nullable.